src/test/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSyncSpec.scala

```diff
@@ -92,7 +92,10 @@ class RegularSyncSpec
         regularSync ! SyncProtocol.Start
 
         peerEventBus.expectMsgClass(classOf[Subscribe])
-        peerEventBus.reply(MessageFromPeer(NewBlock(testBlocks.last, testBlocks.last.number), defaultPeer.id))
+        // It's weird that we're using block number for total difficulty but I'm too scared to fight this dragon
```

Comment:
😂

Reply:
This test continues to be the most difficult one to follow, at least among the ones I've dealt with recently